scripts/generate

```diff
@@ -159,6 +150,8 @@ def('doctestsToMarkup',
     [$.String, $.String],
     pipe([matchAll(/^> .*(?:\n[.] .*)*/gm),
           map(prop('match')),
+          map(replace(/\[\n[.] */g, '[')),
+          map(replace(/,\n[.] \]/g, ']')),
```

Comment:
This prevents wrapped array literals becoming `[ elem1, elem2, elem3,]` when joined.